testing/test_process_result.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from manageiq_client.api import Collection, ActionContainer, Action
+
+
+url_bases = [
+    "https://example.com/api/v2.4.0/",
+    "https://example.com/api/",
+    "https://127.0.0.1:8080//api",
```

Comment:
@mkoura do we support also plain http? Do you want to add a test to make sure it can be used?